module/nilai.go

```diff
@@ -0,0 +1,54 @@
+package module
+
+import (
+	"context"
+
+	nilai "github.com/tokopedia/code-review-workshop/core/nilai"
+	"github.com/tokopedia/tokopoints/errors"
+	"github.com/tokopedia/tokopoints/tracer"
+)
+
+func GetNilaiSiswa(ctx context.Context, nim int64, sekolah, kelas string, mapel_id int64) (resp GetNilaiSiswaResponse, err error) {
+	span, ctx := tracer.StartSpanFromContext(ctx)
+	defer span.Finish()
+
+	if nim != 0 && mapel_id != 0 {
```

Comment:
ini if nya dijadiin switch case aja
```suggestion
	switch b := param; {
	case b.NIM > 0 && b.MapelID > 0:
		fmt.Println("BLA")
	case b.Sekolah == "SekolahGlobal":
		fmt.Println("Semua")
	default:
		fmt.Println("ga semua")	
	}
```